components/admin_console/data_table/data_table.tsx

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import LoadingSpinner from 'components/widgets/loading/loading_spinner';
+
+import DataTableHeader from './data_table_header';
+import DataTableRow from './data_table_row';
+
+import 'components/next_steps_view/next_steps_view.scss';
+
+import './data_table.scss';
+
+export type Column = {
+    name: string | JSX.Element;
```

Comment:
There's a type called `React.ReactNode` you can use here instead of `string | JSX.Element`. It covers a lot more bases of what can be rendered.